src/styles/components/_confirmation.scss

```diff
@@ -0,0 +1,60 @@
+.background-lion {
+  background-size: cover;
+  background-repeat: no-repeat;
+  -webkit-background-size: contain;
+  -moz-background-size: contain;
+  -o-background-size: contain;
+  height: 420px;
+  padding: 45px 0 0 60px;
+  margin: 0;
+  display: grid;
+  grid-gap: 15px 5px;
+  grid-template-columns: 30% 20%;
+  grid-template-rows: 24% 38% 20%;
+  // small/mid
+  // grid-template-columns: 50% 32%;
+  // grid-template-rows: 20% 36% 20%;
```

Comment:
Did you mean to get rid of the commented-out lines?

Reply:
No. They're for the next responsive step.